sdk/keyvault/keyvault-keys/samples/v4/javascript/helloWorld.js

```diff
@@ -53,6 +53,7 @@ async function main() {
   });
   console.log("updated key: ", updatedKey);
 
+  // Delete the keys
```

Comment:
Thank you for this update! The system we use today relies on source code defined in https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-keys/samples-dev/helloWorld.ts

It looks like we never published the recent changes to our samples - I can open a PR to do that (and I'll ensure comments are present).

But just as an FYI - making a change in the samples/ folder will likely be overwritten whenever someone "publishes" the `samples-dev` samples - so any changes should be made there.

Here's some more information about our new sample flow https://github.com/Azure/azure-sdk-for-js/wiki/Samples-v2-Migration-Guide

Reply:
@maorleger I think the secret's hello world would benefit from the same delete poller code. If that is acceptable to you, can I submit a PR or open an issue?

Reply:
Absolutely! Feel free to either submit a PR or open an issue and assign to me.